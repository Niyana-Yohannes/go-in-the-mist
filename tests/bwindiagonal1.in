place A19
place A1
place B18
place B1
place C17
place C1
place D16
place D1
place E15
