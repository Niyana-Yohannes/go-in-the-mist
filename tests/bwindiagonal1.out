Black wins!
A19A1B18B1C17C1D16D1E15
Thank you for playing!
